### PR TITLE
Update url displayed in navbar when links are clicked

### LINKED ISSUE #51: Don't update url in NavBar when movement page
**Do you want to request a *feature* or report a *bug*?**
feature

**What is the current behavior?**
Url in NavBar is not updated when movement page like click the link.

**Expected behavior**
Url in NavBar will change when movement page.

Is this intended?

### PR DESCRIPTION
**Pull request by melanieseltzer**
**Related Issue**
Fixes #51

When the user clicks on links in the browser, the url in the navbar will now update accordingly. The link clicks are captured using the `did-navigate` DOM event.

Note: When entering a url that is without protocol (e.g. github.com) it will now first resolve to http://github.com and then finally to https://github.com/ but I'm not sure why this is. Also seems to now force a trailing slash. You can see this behavior in the capture below, along with the new url update behavior.

### CODE REVIEW

File: src/components/web-page/index.js [hunk 1]
```diff
@@ -51,6 +51,13 @@ class WebPage extends React.Component {
         this.props.onUrl(newUrl);
       }
     });
+
+    // Capture link clicks on page and update state with new url
+    currentWebView.addEventListener('did-navigate', (event) => {
+      this.setState({
+        url: event.url
+      });
+    });
```
Comment (HyunmoAhn) [inline src/components/web-page/index.js]: I know that need to more code about update url.

**`did-navigate` event doesn't cover [SPA(Single Page Application)](https://en.wikipedia.org/wiki/Single-page_application) Page redirect like using `react-router`.**

So Must use `did-navigate-in-page` event.

Following sentence is [document of `did-navigate-in-page`](https://electronjs.org/docs/api/webview-tag#event-did-navigate-in-page)
> When in-page navigation happens, the page URL changes but does not cause navigation outside of the page. 

## Example 
If you enter [sample page](https://app.modusign.co.kr), you may be enter `https://app.modusign.co.kr` but webpage redirect you from `https://app.modusign.co.kr` to `https://app.modusign.co.kr/authentication/signin` . You can check in normal-browser like Chrome. If you use only `did-navigate` event, url doesn't change to `https://app.modusign.co.kr/authentication/signin`

## Recommend
I recommend to write more code without deleting current code.

```
    currentWebView.addEventListener('did-navigate-in-page', (event) => {
      this.setState({
        url: event.url
      });
    });
```

**Comment by HyunmoAhn** [comment]
> When entering a url that is without protocol (e.g. github.com) it will now first resolve to http://github.com and then finally to https://github.com/ but I'm not sure why this is.

Because When enter url without `http(s)?//` format in empty page, `prepareUrl` util function convert url from `example.com` to `http://example.com` forcefully.

https://github.com/kamranahmedse/pennywise/blob/master/src/utils/helpers.js#L114

File: src/components/web-page/index.js (hunk 1 above)
Comment (melanieseltzer) [reply src/components/web-page/index.js]: Thank you 👍 Have added it to PR.

**Comment by kamranahmedse** [comment]
Thanks

**Pull request merged**